docker-compose.yml

```diff
@@ -0,0 +1,66 @@
+# To execute this docker-compose yml file use `docker-compose -f <file_name> up`
+# Add the `-d` flag at the end for detached execution
+version: "3"
+volumes:
+  my-datavolume:
+  redis-data:
+  esdata1:
+services:
+  db:
+    image: mysql:5.7
+    container_name: otwa_db
+    environment:
+      - MYSQL_ROOT_PASSWORD=change_me
+    ports:
+      - "3306:3306"
+    command:
+      [
+        "mysqld",
+        "--sql-mode=STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION",
+      ]
+    volumes:
+      - my-datavolume:/var/lib/mysql:rw
+  redis:
+    image: redis:5
+    container_name: otwa_redis
+    ports:
+      - "6379:6379"
+    volumes:
+      - redis-data:/var/lib/redis:rw
+  es:
+    image: docker.elastic.co/elasticsearch/elasticsearch:6.8.9
+    container_name: otwa_es
+    ports:
+      - "9200:9200"
+      - "9300:9300"
+      - "9400:9400"
+    volumes:
+      - esdata1:/usr/share/elasticsearch/data:rw
+    environment:
+      - transport.host=localhost
+      - bootstrap.memory_lock=false
+      - "ES_JAVA_OPTS=-Xms1500m -Xmx1500m"
+      - discovery.type=single-node
+  mc:
+    image: memcached:1.5
+    container_name: otwa_mc
```

Comment:
Is there an advantage to having the container name, I just tried to bring two instances up so I could run cucumber tests and rspec at the same time and failed due to having a duplicate container name ?

Reply:
No, I don't think there is any advantage here. I don't use `container_name` in any of my other work. Will remove.